.github/ISSUE_TEMPLATE/spec_request.md

```diff
@@ -0,0 +1,108 @@
+* make sure that YAML blocks contain well-formed YAML;
+* make sure that blocks contents do not violate their schemas;
+* make sure that all the fields marked as MANDATORY are filled in.
+
+-->
+```yaml
+# The purposes of this block are:
+# * storing the info uniquily identifying the format to make it searcheable;
+# * storing useful information about the format in a convenient structured way;
+# * providing an implementer with a boilerplate which can be just copied into a KSY file;
+# The content of this block must be a part of a valid KSY.
+# This block must be present.
+
+meta: # see https://doc.kaitai.io/ksy_reference.html#meta for more info
+  id: # insert an ID (matches the regexp [a-z][\da-z]*) you feel right to assign to the format here. KS currently has no namespaces, so the id MUST be unique within all the repo and you should be sure there will be no collisions. If in doubt, add a prefix of an app name. MANDATORY.
+  title: # insert a human-readable name of the format here. MANDATORY.
+  application: # insert a human-readable name of the application for which this format is native here. If it is native for several applications, choose the ones you feel the most affiliated with the format. If the format is a widespread one, like PNG or WAV, just remove this field.
+  file-extension:
+    # insert here a YAML list of file extensions should be placed here.
+  xref: # check if the following sources contain info about the format you request. See https://doc.kaitai.io/user_guide.html#meta-xref for more info
+    din:
+      id: # DIN standard identifier
+    forensicswiki: # Page name on https://forensicswiki.org/
+    fileformat: # Link to https://www.fileformat.info
+    iso: # ISO standard identifier
+    justsolve: # ID on http://fileformats.archiveteam.org/wiki/
+    loc: # ID in https://www.loc.gov/preservation/digital/formats/
+    mime: # MIME type
+    pronom: # IDs on https://www.nationalarchives.gov.uk/pronom/fmt/
+     - x-fmt/
+    rfc: # Link to RFC
+    wikidata: Q # ID on WikiData
+
+doc: |
+  # Describe your format briefly AND add links to the sources of sample files in the Web. MANDATORY
+
+doc-ref:  # A YAML array of URIs to information sources and/or their names goes here. MANDATORY. See https://doc.kaitai.io/ksy_reference.html#doc-ref for more info on sources which are not just an URI
+  - ref 1
+  - ref 2
+  - ref 3
+  # There are at least 3 kinds of sources complementing each other.
+  # * Official specs.
+  #    * Pros:
+  #      * they are official
+  #      * they usually conduct the the meaning by the authors of the format.
+  #    * Cons:
+  #      * often obsolete and incomplete - don't document extensions by third-parties, even de-facto standards, met in the wild
+  #      * often they are licensed under a proprietary license, so it may be illegal to quote them extensively into `doc`.
+  #    * How to find:
+  #      * usually in PDF, HTML and RTF, Google dork filetype:pdf may be useful
```

Comment:
What is "Google dork"?

For finding *official* specs it's usually easier to check the website of the format (or its main application/company) or the format's Wikipedia page. That way you can be relatively certain that the spec you find is really official and current.

Reply:
>What is "Google dork"?

A search query for Google search engine targeted to find specific information using its search operators.

>For finding official specs it's usually easier to check the website of the format (or its main application/company) or the format's Wikipedia page. That way you can be relatively certain that the spec you find is really official and current.

Sometimes - yes. Sometimes the official spec is buried inside vendor website and the vendor has lot of specs, or the spec is for an obsolete format, so its pdf is removed from the offsite and available only via third-parties.